### ISSUE TITLE
Ensure map search works as expected

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#938]
**URL:** http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/dac2020/3-a-1/

**Screen shot:**
![image](https://user-images.githubusercontent.com/43404358/94694573-e62bb300-032c-11eb-8bc4-dae20e2c2cb9.png)

The search ‘button’ (implemented as a link) performs two functions. On the first instance it is to trigger the search input to appear. However, when the ‘search’ icon is focused and enter is pressed, the search input flashes onto the screen and then disappears again. There is nothing to relay its collapsed or expanded status to screen reader users.
Once the search input is expanded, if there is text in the search input, the search ‘button’ link then acts as a submit button allowing users to submit their search. If there is no text within the search field, it collapses the search field. This behaviour is not described to screen reader users.

**Solution:**
Please ensure that all functionality is able to be operated using the keyboard. To allow screen reader users to understand that a link triggers content to appear, an aria-expanded = “false” attribute can be used when the button is in a collapsed state, and a descriptive label should be provided such as aria-label="show search". When this button is configured to collapse the component, an aria-expanded = “true” attribute can be used with an aria-label="hide search". When this button is used as a search button, it should have no aria-expanded attribute and an aria-label="search".

**Comment by brockfanning** [comment]
Hopefully this third-party library has already been updated, as this seems like a big problem. But I'll investigate and override if needed.
